src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("\fonts\Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url("\fonts\Avenir-Heavy.ttf");
+}
+
+.Search {
+  margin-top: 20px;
+}
+
+.SearchBar {
+  position: relative;
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.SearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.SearchBar:active,
+.SearchBar:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 3px;
+}
+
+.SearchBar--big {
+  height: 70px;
+}
+
+.SearchBar--small {
+  height: 42px;
+}
+
+.SearchBar__icon {
+  display: block;
+  position: absolute;
+  pointer-events: none;
+}
+
+.SearchBar--big .SearchBar__icon {
+  top: 24px;
+  left: 25px;
+}
+
+.SearchBar--small .SearchBar__icon {
+  width: 11px;
+  height: 11px;
+  top: 14px;
+  left: 12px;
+}
+
+.SearchBar__input {
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
```

Comment:
should be `300`, move `900` to `:focus::placeholder`

Reply:
@ivanchuk-yurii, why should I move the `font-weight: 900;`  from  `.SearchBar__input ` to  @`.SearchBar__input:focus::placeholder`?
`.SearchBar__input ` selector stylizes input value text, at the same time  `.SearchBar__input:focus::placeholder` stylizes only placeholder when input has focus state.
In mockup search text is bold when typing a search value...